.github/workflows/brim-zeek-wrapper

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
```

Comment:
Would you please move this script into the top level `brim` folder I mentioned in my earlier comment?

Reply:
👍